### ISSUE TITLE
backgroundColor should be a widget instead of Color

### ISSUE THREAD

**Issue by huteri** [Codelessly/ResponsiveFramework#39]
```
======== Exception caught by widgets library =======================================================
The following _TypeError was thrown building ResponsiveWrapper(dirty, state: _ResponsiveWrapperState#c9032):
type 'Color' is not a subtype of type 'Widget'

The relevant error-causing widget was: 
  ResponsiveWrapper file:///Users/huteri/flutter/.pub-cache/hosted/pub.dartlang.org/responsive_framework-0.0.14/lib/responsive_wrapper.dart:119:12
When the exception was thrown, this was the stack: 
#0      _ResponsiveWrapperState.build (package:responsive_framework/responsive_wrapper.dart:484:5)
#1      StatefulElement.build (package:flutter/src/widgets/framework.dart:4744:28)
#2      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4627:15)
#3      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4800:11)
#4      Element.rebuild (package:flutter/src/widgets/framework.dart:4343:5)
...
====================================================================================================
```

```
 /// First frame initialization default background color.
  /// Because layout initialization is delayed by 1 frame,
  /// a solid background color is displayed instead.
  /// Default white.
  final Color backgroundColor;
```

Used here as widget instead of color 
```
return (screenWidth ==
            0) // Initialization check. Window measurements not available until postFrameCallback.
        ? (widget.backgroundColor ??
            Container(
                color: Color(
                    0xFFFFFFFF))) // First frame empty background color or default white.
        : InheritedResponsiveWrapper(
```

**Comment by searchy2** [comment]
Thank you for catching this error!

**Comment by huteri** [comment]
You are welcome @searchy2. Any plan to release this fix?

**Comment by searchy2** [comment]
I would like to batch the fix with some other fixes for a package release.
I'll push a fix for this bug and you can use it directly via Git import in a bit.

**Comment by searchy2** [comment]
Fixed via https://github.com/Codelessly/ResponsiveFramework/commit/c55ab5c50ff0deeec2fc633d19a159a8f0e394eb

For now, you can use the fix on the latest branch with:
responsive_framework:
    git:
      url: https://github.com/Codelessly/ResponsiveFramework.git
      ref: master

**Comment by huteri** [comment]
thanks @searchy2 appreciate it.

**Comment by SdxCoder** [comment]
When the fix will be pushed to pub dev. Need to use this in a production app.

**Comment by searchy2** [comment]
@SdxCoder I'm fixing one more bug and then I'll release a fix. 
Thank you for your patience.

**Comment by searchy2** [comment]
Fix available in v0.0.15
